projects/packages/jetpack-mu-wpcom/src/features/wpcom-profile-settings/profile-settings-link-to-wpcom.ts

```diff
@@ -1,95 +1,140 @@
-/**
- * Disable the email field except on Atomic Classic sites.
- */
-const wpcom_profile_settings_disable_email_field = () => {
-	if ( window.wpcomProfileSettingsLinkToWpcom?.isWpcomAtomicClassic ) {
-		return;
+const wpcom_profile_settings_modify_language_section = () => {
+	const section = document.querySelector( '.user-language-wrap' )?.querySelector( 'td' );
+	const select = document.getElementById( 'locale' );
+	const settingsLink = window.wpcomProfileSettingsLinkToWpcom?.language?.link;
+	const settingsLinkText = window.wpcomProfileSettingsLinkToWpcom?.language?.text;
+	if ( settingsLink && settingsLinkText ) {
+		const notice = document.createElement( 'p' );
+		notice.className = 'description';
+		notice.innerHTML = `<a href="${ settingsLink }">${ settingsLinkText }</a>`;
+		section?.appendChild( notice );
+		select?.remove();
 	}
-	const emailField = document.getElementById( 'email' ) as HTMLInputElement;
-	if ( emailField ) {
-		emailField.readOnly = true;
+};
+
+const wpcom_profile_settings_modify_name_section = () => {
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg) Is the `Username` field equivalent with dotcom's Username in /me/account? 
(If so, should we display the `Username` field and link to /me/account?)

Reply:
Interesting... yes, it's equivalent.

Then how should we put the link to `/me` 😂 I don't want to put the same thing individually to `First Name`, `Last Name` etc... let me think about it a bit more...

Reply:
After contemplating, I decided to keep this PR as-is:

- changing username is not allowed in Core anyway,
- user can check our help center if they want to change wpcom username, and
- I don't think it's worth it to add yet another link for a very very rare scenario (changing username) 😄 

Let me know if you strongly object!

Reply:
Would it make sense to have "Name" for /me and "Username" for /me/account? 
I wanted to address that only the ”Username" has a setting on a different page than /me.

Reply:
Oh...... let me try!

Reply:
Oh, I missed your previous comment!

Reply:
I'm good leaving the decision to you. 🙂

Reply:
No worries! I just tried the above idea but the IA becomes awkward; having a 2x2 table with heading at top left 😅 

I'll just go head merging this and ask for forgiveness! 🙏